gammapy/catalog/tests/test_hess.py

```diff
@@ -21,6 +23,31 @@ def test_component_table(self):
     def test_associations_table(self):
         assert len(self.cat.associations) == 223
 
+    def test_large_scale_component(self):
+        # This test compares the flux values from the LS model within source
+        # regions with ones listed in the catalog, agreement is <1%
+        ls_model = self.cat.large_scale_component
+
+        for name in ['HESS J1837-069', 'HESS J1809-193', 'HESS J1841-055']:
```

Comment:
Use `pytest.parametrize` instead of for loop to make the test function a bit shorter and less indented?
(and if one fails, the error message will be better)